### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by lisazeyen**
- add missing color for H2 liquidation to config.default.yaml
- adjust for myopic optimisation selection of assets to new default lifetime and build year
- remove trailiing whitespace in index of electricity load of pypsa-eur network

### CODE REVIEW

File: scripts/prepare_sector_network.py [hunk 1]
```diff
@@ -270,6 +270,9 @@ def patch_electricity_network(n):
     update_wind_solar_costs(n, costs)
     n.loads["carrier"] = "electricity"
     n.buses["location"] = n.buses.index
+    # remove trailing white space of load index
+    n.loads.rename(lambda x: x.strip(), inplace=True)
+    n.loads_t.p_set.rename(lambda x: x.strip(), axis=1, inplace=True)
```
Comment (fneum) [inline scripts/prepare_sector_network.py:275]:
```suggestion
    # remove trailing white space of load index until new PyPSA version after v0.18.
    n.loads.rename(lambda x: x.strip(), inplace=True)
    n.loads_t.p_set.rename(lambda x: x.strip(), axis=1, inplace=True)
```

File: scripts/prepare_sector_network.py [hunk 2]
```diff
@@ -2007,10 +2010,11 @@ def limit_individual_line_extension(n, maxext):
         snakemake = mock_snakemake(
             'prepare_sector_network',
             simpl='',
-            clusters=48,
+            clusters='45',
```
Comment (fneum) [inline scripts/prepare_sector_network.py:2013]:
```suggestion
            clusters='45,
```

File: scripts/prepare_sector_network.py [hunk 3]
```diff
@@ -2007,10 +2010,11 @@ def limit_individual_line_extension(n, maxext):
...
             lv=1.0,
+            opts='',
             sector_opts='Co2L0-168H-T-H-B-I-solar3-dist1',
-            planning_horizons=2020,
+            planning_horizons='2030',
```
Comment (fneum) [inline scripts/prepare_sector_network.py:2017]:
```suggestion
            planning_horizons=2030,
```

**Review by fneum — APPROVED** [review]
Just one comment on the `lstrip` about when to remove it again.

File: scripts/prepare_sector_network.py [hunk 4]
```diff
@@ -2007,10 +2010,11 @@ def limit_individual_line_extension(n, maxext):
         snakemake = mock_snakemake(
             'prepare_sector_network',
             simpl='',
-            clusters=48,
+            clusters='45,
```
Comment (fneum) [inline scripts/prepare_sector_network.py:2013]:
```suggestion
            clusters=45,
```

sorry, that commit was a bit odd.

**Comment by fneum** [comment]
Sorry, made a big mess squashing this time since it's only a few lines and doesn't contribute to stats.

File: scripts/prepare_sector_network.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (lisazeyen) [reply scripts/prepare_sector_network.py]: No, it does not. It breaks if it is of type `int` when slicing in the function [update_wind_solar_costs](https://github.com/PyPSA/pypsa-eur-sec/blob/7d0ff39b1a1fdf19101b51fc3adc0ff112210ac3/scripts/prepare_sector_network.py#L321)

**Pull request merged**